src/items/DownloadButtonFileItem.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles(() => ({
+  link: {
+    textDecoration: 'none',
+  },
+}));
+
+const DownloadButtonFileItem = ({ id, name, url }) => {
+  const classes = useStyles();
+
+  return (
+    <a
+      id={id}
+      className={classes.link}
+      href={url}
+      target='_blank'
+      rel='noreferrer'
+      download={name}
+    >
+      <Button
+        color='primary'
+        variant='contained'
+        size='large'
+        startIcon={<GetAppIcon />}
+      >{`Download ${name}`}</Button>
```

Comment:
Do we not have translation in UI? Or are we leaving that to each implementation?

Reply:
We don't have translation, I think it is better to let the user provide a string, otherwise it defaults to english/set text. ideally it would be nice to provide all languages and depends it on the implementation, but it would require to export an instance of `i18n` and I'm not sure it is easy to extends with multiple language files... It might be interesting to investigate at some point.